src/fontra/client/lang/zh-CN.json

```diff
@@ -86,6 +91,8 @@
   "menubar.file.open": "打开……",
   "menubar.font": "字体",
   "menubar.font.edit": "编辑字体信息、参数轴和源",
+  "menubar.fontra": "字挼 (zìruá)",
```

Comment:
Where did this translation come from?

Reply:
Here: https://github.com/googlefonts/fontra/issues/1461

Reply:
Ah, I never closed this issue, but we decided back then (with Mark and other Chinese speaker at Black Foundry) that we'll not go with that name. It was even suggested the issue may be trolling.

Reply:
OK, thanks. I was not aware of that. I removed the translation.